zebrad/src/commands/generate.rs

```diff
@@ -31,8 +31,20 @@ impl Runnable for GenerateCmd {
 # can be found in Rustdoc here:
 # https://doc.zebra.zfnd.org/zebrad/config/struct.ZebradConfig.html
 
+# Usage:
+# This file must be located in the same directory the zebrad binary is
```

Comment:
AFAIK the config doesn't have to be in the same dir, it's just that it has to either be passed in via `-c` _or_ it has to be in the same directory to be found automatically.

Reply:
Seems you are right, thanks. Changed the text to reflect that.

Reply:
Apologies, i pushed without committing the changes. Now pushed again with the changes.